src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/migration/MigrateVideoBookingService.kt

```diff
@@ -43,7 +43,7 @@ class MigrateVideoBookingService(
 
   private fun migrateCourt(bookingToMigrate: VideoBookingMigrateResponse): VideoBooking {
     val prisonerNumber = mappingService.mapBookingIdToPrisonerNumber(bookingToMigrate.offenderBookingId)
-      ?: throw MigrationException("Unable to find prisoner number for booking ${bookingToMigrate.videoBookingId}")
+      ?: throw MigrationException("Unable to find prisoner number for offender booking ID ${bookingToMigrate.offenderBookingId} for court booking ${bookingToMigrate.videoBookingId}")
```

Comment:
Saves having to look up the video booking, now has all we need to check i.e. the offender booking id.